spec/support/tests/rails_versions.rb

```diff
@@ -1,8 +1,8 @@
-module UnitTests
+module Tests
   module RailsVersions
     extend self
 
-    def self.configure_example_group(example_group)
+    def configure_example_group(example_group)
```

Comment:
Heads up that the `self.` was intentional to communicate that `configure_example_group` is callable at the class level but shouldn't be available within tests.

Reply:
I see.

To be honest with you I did not know much about `extend self` until I saw this file. After reading about it I came to the conclusion that the `self.` was not necessary. I thought it was redundant. My bad. Thanks for the heads up.

I'll revert this change.